src/UseCases/AddComment/AddCommentUseCase.php

```diff
@@ -46,6 +51,10 @@ public function addComment( AddCommentRequest $addCommentRequest ): AddCommentRe
 
 		$donation->addComment( $this->newCommentFromRequest( $addCommentRequest ) );
 
+		if ( !$this->commentTextPassesValidation( $addCommentRequest->getCommentText() ) ) {
```

Comment:
Right. The old application [did](https://github.com/wmde/fundraising/blob/master/src/CommentHandler.php#L42), though. As far as I remember, we introduced that a while back, because there were false positives produced by punctuation mistakes. What looks like a URL doesn't necessarily have to be [one.is](one.is) what I mean.